src/containers/BooksForm.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { actionCreate } from '../actions/index';
+
+export class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      input: '',
+      select: 'Action',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      input: event.target.value,
+    });
+  }
+
+  handleSelect(event) {
+    this.setState({
+      select: event.target.value,
+    });
+  }
```

Comment:
According to the instruction on pathwright;

we need to make use of `handleChange` method to handle both `input changes` and `select category`.

![image](https://user-images.githubusercontent.com/31897434/96936548-332f1f00-14be-11eb-875c-a33587ae4de0.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896828/

i.e we could have it this wat;

```
handleChange(event) {
  const name = event.target.name;

  this.setState({
    title: event.target.value
    [name]: event.target.value
  })
}
```

**OR**

```
handleChange(event) {
  this.setState({
    input: event.target.value
    select: event.target.value
  })
}
```